nstar/src/main.rs

```diff
@@ -200,32 +200,89 @@ stop <name>:    Stop application"
         .into()
 }
 
-fn run_cmd<S: Read + Write>(cmd: &str, stream: S) -> Result<Option<Response>> {
+fn run_cmd<S: Read + Write>(cmd: &str, stream: &mut S) -> Result<Option<Vec<Response>>> {
```

Comment:
Is the optional really needed in the return type? does nstar make a distinction between a None and Some(vec![])?

Reply:
The None is originally used to report that nothing was done. Probably this is more an error because the input didn't lead to any action. Will rethink this.